addons/openebs/3.3.0/install.sh

```diff
@@ -384,8 +384,18 @@ function openebs_prompt_migrate_from_longhorn() {
         bail "Not migrating"
     fi
 
+    if [ -z "$MINIO_VERSION" ] ; then
+        logFail "    You can only migrate from Longhorn to OpenEBS if you are specifying Minio as your Object Store."
+        logFail "    Please make sure you are including Minio in your kURL Installer spec and try again."
+        bail "Not migrating"
+    fi
```

Comment:
what if they have no object store at all?

Reply:
That is a good point.

Reply:
Does that mean "no kots" ?

Reply:
<img width="457" alt="Screenshot 2023-02-13 at 18 50 51" src="https://user-images.githubusercontent.com/1385436/218534255-9c6b8bb0-4f98-42ec-be69-d73687f36c2d.png">

Reply:
You can have kots with no object store 😄

Reply:
theoretically we could have a kurl install without kots. 
In the reality I think you are right :-)

In this way, would be possible:

- warning instead of bail
- in the waning suggest add minio and say what are the implications
- confirm if they want or not continue to ensure
- if they continue without minio skip the specific step and just migrate ?

Reply:
There is a flag (in kURL) to run kots with no object store

Reply:
I have removed the check for Minio. My reasoning is that if the cluster is running Kots+Longhorn without Minio then a simple PVC migration will be enough. I will test this to make sure it works.

Reply:
Tested with Kots' `disableS3` option and it works (after removing the Minio presence check introduced by this PR).